.pre-commit-config.yaml

```diff
@@ -10,3 +10,13 @@ repos:
     hooks:
     -   id: black
         exclude: ^docs/
+-   repo: local
+    hooks:
+    -   id: freeze-dependencies
+        name: Freeze Dependencies
+        entry: python -m freeze_dependencies
+        language: system
+        pass_filenames: false
```

Comment:
This is a pretty interesting idea; could you open an issue+PR on the cookiecutter template for future discussion to see what the rest of the team things?